### ISSUE TITLE
[RegisterSoftware] FTC interrupts Copyright Office probe to flip out over potential AI fraud, abuse

### ISSUE THREAD

**Issue by hinoshiba** [hinoshiba/news#5744]
#### Sir, this is a Wendy's

AI hypesters may think their training datasets and models' output are or ought to be protected from copyright claims, but their neural networks could still fall foul of consumer protection laws, the FTC has has warned.…

<https://go.theregister.com/feed/www.theregister.com/2023/11/09/ftc_ai_regulation/>

**Comment by hinoshiba** [comment]
This issue is stale because it has been open 1 day with no activity. Remove stale label or comment or this will be closed in 1 day.

**Comment by hinoshiba** [comment]
This issue was closed because it has been stale with no activity.
